Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/Sharing/MarkerDetectorLocalizationSettings.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public class MarkerDetectorLocalizationSettings : ISpatialLocalizationSettings
+    {
+        public int MarkerID { get; set; }
+        public float MarkerSize { get; set; }
+
```

Comment:
Can these settings also contain the marker detector type? If these settings could specify what marker detector to use or a game object prefab to create that contains an IMarkerDetector, we may be able to enable more customization?

Reply:
this seems beyond whats needed for now.